.github/workflows/build.yml

```diff
@@ -35,8 +35,16 @@ jobs:
             *.cache-from=type=gha,scope=build
             *.cache-to=type=gha,scope=build,mode=max
 
+  vale:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@v2.1.0
```

Comment:
Should pin to commit sha for unofficial actions

```suggestion
      - uses: errata-ai/vale-action@38bf078c328061f59879b347ca344a718a736018  # v2.1.0
```

Reply:
thanks, yeah good catch